package.json

```diff
@@ -27,7 +27,8 @@
   ],
   "dependencies": {
     "request": "latest",
-    "debug": "~0.7.2"
+    "debug": "~0.7.2",
+    "mustache": "^2.2.1"
```

Comment:
Also : underscore is already there and already contains a [templating utility](http://underscorejs.org/#template). Might have been sufficient here.